### PR TITLE
MoreMMR Integration

### PR DESCRIPTION
**Pull request by albertcui**
Added button: http://albertcui.com:8080/matches/3840943814/overview
Added heroes: http://albertcui.com:8080/heroes
Added logo: http://albertcui.com:8080/

### CODE REVIEW

File: src/components/Match/matchColumns.jsx [hunk 1]
```diff
@@ -33,8 +33,9 @@ export const heroTd = (row, col, field, index, hideName, party, showPvgnaGuide =
     confirmed={row.account_id && row.name}
     party={party}
     heroName={heroes[row.hero_id] ? heroes[row.hero_id].localized_name : strings.general_no_hero}
-    showPvgnaGuide={showPvgnaGuide}
-    pvgnaGuideInfo={row.pvgnaGuide}
+    showGuide={showPvgnaGuide}
```
Comment (howardchung) [inline src/components/Match/matchColumns.jsx]: shouldn't you just pass all of these in as props?

File: src/components/Visualizations/Table/HeroImage.jsx [hunk 2]
```diff
@@ -348,13 +349,15 @@ const TableHeroImage = ({
...
       }
-      { !!showPvgnaGuide && pvgnaGuideInfo && heroName &&
-      <div className="pvgnaGuideContainer" data-tip data-for={heroName}>
-        <a href={pvgnaGuideInfo.url}>
-          <img className="pvgnaGuideIcon" src="/assets/images/pvgna-guide-icon.png" alt={`Learn ${heroName} on Pvgna`} />
+      { !!showGuide && guideType && guideUrl && heroName &&
+      <div className="guideContainer" data-tip data-for={heroName}>
+        <a href={guideUrl}>
+          { guideType === 'PVGNA' ? <img className="guideIcon" src="/assets/images/pvgna-guide-icon.png" alt={`Learn ${heroName} on Pvgna`} /> : <div /> }
+          { guideType === 'MOREMMR' ? <img className="guideIcon" src="/assets/images/moremmr-icon.png" alt={`Learn ${heroName} on MoreMMR`} /> : <div /> }
```
Comment (howardchung) [inline src/components/Visualizations/Table/HeroImage.jsx]: the alt text should prob be localized

File: src/components/Visualizations/Table/HeroImage.jsx [hunk 3]
```diff
@@ -348,13 +349,15 @@ const TableHeroImage = ({
         }
       </div>
       }
-      { !!showPvgnaGuide && pvgnaGuideInfo && heroName &&
-      <div className="pvgnaGuideContainer" data-tip data-for={heroName}>
-        <a href={pvgnaGuideInfo.url}>
-          <img className="pvgnaGuideIcon" src="/assets/images/pvgna-guide-icon.png" alt={`Learn ${heroName} on Pvgna`} />
+      { !!showGuide && guideType && guideUrl && heroName &&
```
Comment (howardchung) [inline src/components/Visualizations/Table/HeroImage.jsx]: replace !! with Boolean()

File: src/components/Match/matchColumns.jsx (hunk 1 above)
Comment (howardchung) [reply src/components/Match/matchColumns.jsx]: i.e., this should probably be implemented to not be pvgna-specific

Comment (albertcui) [reply src/components/Match/matchColumns.jsx]: This is on match page. I feel like we can update it if necessary.

File: src/utility/index.jsx [hunk 4]
```diff
@@ -274,13 +274,23 @@ export const transformations = {
...
-        pvgnaGuideInfo={row.pvgnaGuide}
+        showGuide={showGuide}
+        guideUrl={guideUrl}
+        guideType={guideType}
         leaverStatus={row.leaver_status}
       />
     );
   },
-  hero_id_with_pvgna_guide: (row, col, field) => transformations.hero_id(row, col, field, true),
+  hero_id_with_pvgna_guide: (row, col, field) => transformations.hero_id(row, col, field, true, IMAGESIZE_ENUM.SMALL, row.pvgnaGuide ? row.pvgnaGuide.url : null, 'PVGNA'),
```
[the remark this answers is not in the document]
Comment (albertcui) [reply src/utility/index.jsx]: k

[the remark this answers is not in the document]
Comment (albertcui) [reply src/utility/index.jsx]: Does that work? we don't want a boolean.

[the remark this answers is not in the document]
Comment (albertcui) [reply src/utility/index.jsx]: i guess it works.

File: src/utility/index.jsx [hunk 5]
```diff
@@ -274,13 +274,23 @@ export const transformations = {
...
+  hero_id_with_more_mmr: (row, col, field) => {
+    let url = 'https://moremmr.com/en/heroes/';
+    if (heroes[row[col.field]]) {
+      const heroName = heroes[row[col.field]].localized_name.toLowerCase().replace(' ', '-');
```
[the remark this answers is not in the document]
Comment (albertcui) [reply src/utility/index.jsx]: done

File: src/components/Visualizations/Table/HeroImage.jsx (hunk 3 above)
Comment (albertcui) [reply src/components/Visualizations/Table/HeroImage.jsx]: done

File: src/components/Visualizations/Table/HeroImage.jsx [hunk 6]
```diff
@@ -384,11 +387,12 @@ TableHeroImage.propTypes = {
...
   confirmed: bool,
   heroName: string,
-  showPvgnaGuide: oneOfType([
+  showGuide: oneOfType([
     bool,
     PropTypes.number,
   ]),
-  pvgnaGuideInfo: shape({ url: string }),
+  guideUrl: string,
+  guideType: string,
```
Comment (howardchung) [inline src/components/Visualizations/Table/HeroImage.jsx]: optional: you could probably infer the guideType from the Url if you want to save a prop

**Pull request merged**